turtle-graphics/stretch.js

```diff
@@ -0,0 +1,154 @@
+class Turtle{
```

Comment:
You could add the logic for stretch in the same turtle script rather than creating a replica. Keep your code dry.